src/views/router_views/individualPscList/individualPscList.njk

```diff
@@ -0,0 +1,43 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% set title = i18n.individual_psc_list_main_title %}
+{% set dummyRadioButtons = i18n.dummyRadioButtons %}
+{% set dummyPscOption = i18n.dummyPscOption %}
+
+{% block main_content %}
+    <div class="govuk-grid-row">
+
+            <h1 class="govuk-fieldset__legend--l">{{ title }}</h1>
```

Comment:
You don't need this h1 class here you can use the nunjucks marcro do this for you.